docs/sources/source/index.rst

```diff
@@ -78,49 +82,32 @@ then you can simply use pip to install the latest released version (**currently
 
 .. code-block:: bash
 
-    pip install nemo-toolkit  # installs NeMo Core
-    pip install nemo-asr # installs NeMo ASR collection
-    pip install nemo-nlp # installs NeMo NLP collection
-    pip install nemo-tts # installs NeMo TTS collection
+    pip install nemo_toolkit[all] # Installs NeMo Core and all collections including nemo_asr, nemo_nlp, nemo_tts
 
-**Tutorials**
+Tutorials
+---------
 
 * `Speech recognition <https://nvidia.github.io/NeMo/asr/intro.html>`_
```

Comment:
Shouldn't we use RST/internal links instead?

Reply:
You're probably right. Let me see if I can amend this

Reply:
updated